resolwe_bio/processes/import_data/pathway_map.yml

```diff
@@ -0,0 +1,34 @@
+- slug: upload-metabolic-pathway
+  name: Metabolic Pathway File
+  requirements:
+    expression-engine: jinja
+    executor:
+      docker:
+        image: resolwebio/legacy:latest
+    resources:
+      network: true
+  data_name: Metabolic Pathway
+  version: 0.1.0
+  type: data:pathway
```

Comment:
Change to `data:metabolic-pathway`.

Reply:
I think it's not allowed: `data:metabolic-pathway:' does not match '^data:[a-z0-9:]+:$`
Shall I make it camelCase?

Reply:
@jkokosar ?

Reply:
I such cases, I would usually just concat the words without camelCase: `data:metabolicpathway`